### PR TITLE
[RHBA-394] refactor bpmsuite-7 openshift images to rhba-7

### PR DESCRIPTION
**Pull request by errantepiphany**
https://issues.jboss.org/browse/RHBA-394
Signed-off-by: David Ward <dward@redhat.com>

### CODE REVIEW

File: controller/image.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      value: "7.0.0"
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "rhba-controller"
+    - name: "JBOSS_RHBA_CONTROLLER_VERSION"
+      value: "7.0.0"
+    - name: "PRODUCT_VERSION"
+      value: "7.0.0"
+    - name: "ADD_ONS_DISTRIBUTION_ZIP"
+      value: "rhbas-7.0.0.ER1-add-ons.zip"
```
Comment (jakubschwan) [inline controller/image.yaml]: shouldn't be value rhba-7.0.0... (instead of rhbas)?

File: controller/image.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - name: "CONTROLLER_DISTRIBUTION_ZIP"
+      value: "rhbas-7.0-controller-ee7.zip"
```
Comment (jakubschwan) [inline controller/image.yaml]: same as above

File: kieserver/image.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      value: "7.0.0"
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "rhba-kieserver"
+    - name: "JBOSS_RHBA_KIESERVER_VERSION"
+      value: "7.0.0"
+    - name: "PRODUCT_VERSION"
+      value: "7.0.0"
+    - name: "KIE_SERVER_DISTRIBUTION_ZIP"
+      value: "rhbas-7.0.0.ER1-kie-server-ee7.zip"
```
Comment (jakubschwan) [inline kieserver/image.yaml]: same as above

File: smartrouter/image.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    - name: "JBOSS_PRODUCT"
+      value: "rhba-smartrouter"
+    - name: "JBOSS_RHBA_SMARTROUTER_VERSION"
+      value: "7.0.0"
+    - name: "PRODUCT_VERSION"
+      value: "7.0.0"
+    - name: "ADD_ONS_DISTRIBUTION_ZIP"
+      value: "rhbas-7.0.0.ER1-add-ons.zip"
+    - name: "KIE_ROUTER_DISTRIBUTION_JAR"
+      value: "rhbas-7.0-smart-router.jar"
```
Comment (jakubschwan) [inline smartrouter/image.yaml]: same as above

File: controller/tests/features/controller.feature [hunk 5]
```diff
@@ -0,0 +1,10 @@
+@rhba-7/rhba70-controller
+Feature: RHBA Standalone Controller tests
+
+  Scenario: Test REST API is secure
+    When container is ready
+    Then check that page is served
+      | property             | value                 |
+      | port                 | 8080                  |
+      | path                 | /services/rest/server |
```
Comment (jakubschwan) [inline controller/tests/features/controller.feature]: This path is not correct for Standalone Controller. It should look lika (URL):(PORT)/(controller.context)/management/servers
This path is used for the Kie Server

File: controller/image.yaml (hunk 1 above)
Comment (errantepiphany) [reply controller/image.yaml]: This build (rhbas er1) was the most recent one I was told about that is still stable on EAP 7.1. I need a new build before I can update this.

File: controller/image.yaml (hunk 2 above)
Comment (errantepiphany) [reply controller/image.yaml]: My response same as above.

File: kieserver/image.yaml (hunk 3 above)
Comment (errantepiphany) [reply kieserver/image.yaml]: My response same as above.

File: smartrouter/image.yaml (hunk 4 above)
Comment (errantepiphany) [reply smartrouter/image.yaml]: My response same as above.

File: controller/tests/features/controller.feature (hunk 5 above)
Comment (errantepiphany) [reply controller/tests/features/controller.feature]: Thanks for catching this. I will fix it.

**Pull request merged**